client/src/app/appointments/appointments.component.ts

```diff
@@ -20,8 +21,12 @@ export class AppointmentsComponent {
    */
   public onInitialAssessment() {
     console.log('Initial Assessment');
-    Utilities.setSessionStorage('appt', 'Initial Assessment');
-    this.router.navigateByUrl('/ts');
+    this.clearMemory.clearMemory(true, () => {
+      Utilities.setSessionStorage('appt', 'Initial Assessment');
+      this.router.navigateByUrl('/ts');
+    }, () => {
+      this.router.navigateByUrl('/home');
```

Comment:
Optional: If you click outside the modal, you will go home, which is confusing. It would make more sense to do nothing if the user clicks outside or clicks cancel, as long as you add the Audiologist Login button to this page so that there is no need to go home.